### ISSUE TITLE
Allow admin to edit Facility Name

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-reso-admin#380]
## Description
As a BC Parks staff or PO I need to be able to change the name of a facility in the system so that I do not need to ask the development team to do so.

## Acceptance Criteria
* admin user can change name of facility from Daily Parking to Additional Parking

## Notes
* this will likely involved some work with updating/changing the SK so that user can easily change the name and it prevents the issue of user saving an edit on a park and duplicating the park 

## Linked issues
is blocked by
- [x] ~[BRS-794](https://bcparksdigital.atlassian.net/browse/BRS-794)~

**Comment by Dianadec** [comment]
@LindsayMacfarlane  I've now transferred this to the DUP repo.

**Comment by LindsayMacfarlane** [comment]
Can confirm that users can not currently update facility name or facility type. @MarshallHenson - question for you on DUP priorities for this ticket. (note: it looks like this was missed as part of our prioritization session). 
![Image](https://github.com/user-attachments/assets/6501533a-5494-4bc4-aa6d-5d842f39206c)

**Comment by MarshallHenson** [comment]
I don't think we need this prioritized at all. I haven't brought this up in the past; it looks like Kavi added this. 

In fact, I would suggest that we shouldn't ultimately have this control. In a ideal (perhaps utopian, idk) future, the park name and facility/sub-area/etc are pulled from a separate data source (park registry, otherwise).

**Comment by LindsayMacfarlane** [comment]
Thanks for that feedback @MarshallHenson! With that in mind I'm going to close this ticket and mark as a won't fix. If this comes up again in the future, we can create a new issue/ticket.
